### DISCUSSION TITLE
This repository is currently offline.

### DISCUSSION THREAD

**Discussion by mulevcsteam** [community/community#40969] [Repositories]
### Select Topic Area

Bug

### Body

Hello.
I've got a repo thet is 28 days offline.
for this reason i'm not able to delete but iwas able to rename it.

the uri is:
https://github.com/DCPT-KQA/test-DCPT-GITHUB-9fcb67fc5fcf11edae9c796743aaeb41

**Comment by airtower-luna** [comment]
I only get a 404 when trying to access the repository, so I guess it's private. You can try sending a bug report to GitHub support: https://support.github.com/contact/bug-report
